repl/src/main/scala/com/cloudera/livy/repl/SparkInterpreter.scala

```diff
@@ -118,12 +121,36 @@ class SparkInterpreter(conf: SparkConf) extends Interpreter with Logging {
       }
 
       sparkContext = SparkContext.getOrCreate(conf)
-
+      if (conf.getBoolean("spark.repl.enableHiveContext", false)) {
+        try {
+          val loader = Option(Thread.currentThread().getContextClassLoader)
+            .getOrElse(getClass.getClassLoader)
+          if (loader.getResource("hive-site.xml") == null) {
+            warn("livy.repl.enableHiveContext is true but no hive-site.xml found on classpath.")
```

Comment:
This warning is for the case that user want HiveContext but hive-site.xml is not available somehow, such as misconfiguration. I think at least in the interpreter side, it should print some log to indicate whether he use SQLContext or HiveContext.